recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script:
+    - if not exist %SCRIPTS% mkdir %SCRIPTS%
+    - xcopy /v %RECIPE_DIR%\\vs2008_patch\\setup_x64.bat %SCRIPTS%\\setup_x64.bat
+    - xcopy /v %RECIPE_DIR%\\vs2008_patch\\x64 %SCRIPTS%
+    - call %PREFIX%\\Scripts\\setup_x64.bat
```

Comment:
Why are we calling this during the build? We shouldn't be doing this. The build just exists to shuffle these files into a package. Then it is up to the user to run this script when the package is installed. Basically we will run this script as part of `conda-forge-build-setup`.